tests/src/Context/BeanContext.php

```diff
@@ -30,6 +30,13 @@ class BeanContext implements Context {
   protected $random;
 
   /**
+   * The list of block types created before or during a scenario starts.
+   *
+   * @var array
+   */
+  protected $blockType = array();
```

Comment:
I uses the official naming in Bean module context. In bean, bundle is called block type.
Furthermore, bundle is too vague.